Added Dummy data for Main UI choice boxes.

Jzang2011 opened the pull request:
added dummy data for UI presentation. Renamed files to agreed naming convention and proper practices.

Jzang2011 committed:
Added Dummy data for Main UI choice boxes.

michaelplazek commented on MainMenu/src/mainmenu/view/Main.java:
```diff
@@ -6,11 +6,11 @@
   @Override
   public void start(Stage primaryStage) throws Exception {
-    Parent root = FXMLLoader.load(getClass().getResource("mainmenu.fxml"));
+    Parent root = FXMLLoader.load(getClass().getResource("MainMenuView.fxml"));
```
Any reason this is changed? The convention everyone else is using is lowercase module name.

Jzang2011 replied:
This is an FXML file. The naming convention I thought we agreed on was the same as a Class file.

michaelplazek reviewed (changes requested):
These dropdowns need placeholder text - ie. "Select train" or "Select Track" - set as the default value

Jzang2011 committed:
Setting default select values of Choice boxes

Added default items to Choice Boxes and set the default value to those
items.
-Restructured MainMenuView.fxml to accommodate increased width of
choice boxes. Added more containers to handle cases of Node growth,
which will make the view more adaptable.

Jzang2011 committed:
Fixed Checkstyle errors previously missed

-Fixed specified Checkstyle errors.
-changed MainMenuView.fxml to mainmenuview.fxml

michaelplazek reviewed (changes requested):
Lets get the dropdowns the same size.
We also need to recenter everything.
Other than that it'll be good to be merged!

Jzang2011 committed:
Fixed PR issues

-Centered Labels in VBox in the main UI.
-Renamed MainMenuView.fxml to mainmenuview.fxml as per agreed
convention.
-Hardcoded sizes of choice boxes to be the same.

michaelplazek reviewed (approved):
Looks good
